tests/test_utils.py

```diff
@@ -3,9 +3,10 @@
 """
 
 import pytest
+import xarray as xr
```

Comment:
Can you run `isort --profile=black tests/test_utils.py` as the precommit check failed https://github.com/nci/scores/actions/runs/7175823322/job/19539756617

Reply:
done